app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -618,16 +615,35 @@ public boolean onTouch(View v, MotionEvent event) {
      */
     private void createInfoLayout() {
 
-        LayoutInflater inflater = (LayoutInflater) getSystemService(LAYOUT_INFLATER_SERVICE);
-        infoLayout = (LinearLayout) inflater.inflate(R.layout.dialog_info_window, null);
-        layoutParams.gravity = Gravity.CENTER | Gravity.BOTTOM;
```

Comment:
I see. I asked because I wasn't sure this change belongs in this commit.

In general, when you narrow a regression down to a change in a commit that doesn't explain it, it's harder to fix it—was the change clever or stupid? If I fix that back do I introduce some other bug? That's why I feel safer with separate commits, though of course it's a tradeoff.

Here I can't judge whether it's worth it. Was the code relying on `layoutParams` changing? I hope not, haven't checked.